### PR TITLE
feat(pie-css): DSW-1020 add font-size scss mixin

### PR DESCRIPTION
**Pull request by jamieomaguire**
---
"@justeattakeaway/pie-css": minor
---

[Added] - SCSS font-size mixin and scss output testing

### CODE REVIEW

**Review by raoufswe — APPROVED** [review]

File: packages/tools/pie-css/test/scss/unit/validity.spec.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import {
+    describe,
+    it,
+    expect,
+} from 'vitest';
+
+import cssValidator from 'w3c-css-validator';
+import { compileCss } from '../../../utilities/compileCss';
```
Comment (ashleynolan) [inline packages/tools/pie-css/test/scss/unit/validity.spec.ts:8]: Would this path work?

```suggestion
import { compileCss } from './utilities/compileCss';
```

File: packages/tools/pie-css/test/scss/unit/mixins/fontSize.spec.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import {
+    describe,
+    it,
+    expect,
+} from 'vitest';
+
+import { compileCss, stripCSSWhitespace } from '../../../../utilities/compileCss';
```
Comment (ashleynolan) [inline packages/tools/pie-css/test/scss/unit/mixins/fontSize.spec.ts:7]: Same here with the path? Could you use `./…`

**Review by ashleynolan — COMMENTED** [review]
Looks good – small comment on the path but otherwise :shipit:

File: packages/tools/pie-css/test/scss/unit/mixins/fontSize.spec.ts (hunk 2 above)
Comment (sebastienrocks) [reply packages/tools/pie-css/test/scss/unit/mixins/fontSize.spec.ts:7]: Just out of curiosity, are we not able to leverage aliases like with webpack? `@/`, `@icons/`

Comment (jamieomaguire) [reply packages/tools/pie-css/test/scss/unit/mixins/fontSize.spec.ts:7]: I think we do something similar in `tsconfigs` elsewhere in this repo - let me see if I can do it here. We don't use webpack here @sebastienrocks

Comment (jamieomaguire) [reply packages/tools/pie-css/test/scss/unit/mixins/fontSize.spec.ts:7]: will update in the next pr!

**Review by raoufswe — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**